src/llv8-constants.cc

```diff
@@ -179,7 +183,16 @@ void HeapObject::Load() {
 
 
 void Map::Load() {
-  kInstanceAttrsOffset = LoadConstant("class_Map__instance_attributes__int");
+  Error err;
+  kInstanceAttrsOffset =
+      LoadConstant("class_Map__instance_attributes__int", err);
+  if (err.Fail()) {
+    kInstanceAttrsOffset = LoadConstant("class_Map__instance_type__uint16_t");
+    kMapTypeMask = 0xffff;
```

Comment:
Could it be based on V8 version instead?

Reply:
@indutny We could for the moment, but feature detection is better than version detection IMO

Reply:
@bnoordhuis what would you say?

Reply:
That feature detection > version detection. :-)